src-tauri/src/workflow/builtin_facets/output_contracts/review-verdict.md

```
@@ -1,19 +1,19 @@
-You MUST include exactly one `<workflow_output>` block in your response.
+レスポンスに `<workflow_output>` ブロックを必ず1つだけ含めること。
 
-Format:
+フォーマット:
 ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)